corehq/apps/users/models.py

```diff
@@ -2162,6 +2162,18 @@ def get_sql_location(self, domain):
         if loc_id:
             return SQLLocation.objects.get_or_None(domain=domain, location_id=loc_id)
 
+    def get_location_ids(self, domain):
+        return getattr(self.get_domain_membership(domain), 'assigned_location_ids', None)
+
+    @memoized
+    def get_sql_locations(self, domain=None):
+        from corehq.apps.locations.models import SQLLocation
+        loc_ids = self.get_location_ids(domain)
+        if loc_ids:
+            return SQLLocation.objects.get_locations(loc_ids)
+        else:
+            return []
```

Comment:
I recently saw some feedback about this very same thing - the consensus was that this method should return an empty queryset instead of an empty list.  See https://github.com/dimagi/commcare-hq/pull/14019#discussion_r89427760 and https://github.com/dimagi/commcare-hq/pull/14020 for context.